### DISCUSSION TITLE
【求助】我在六分街遇到了一个怪人

### DISCUSSION THREAD

**Discussion by bxzwq** [share121/inter-knot#5698] [公告]
为了方便买录像，我在六分街录像店附近租了一间房，刚出来就被一个自称“风水学大师”的怪人拦住了，说我选的哪都不能选录像店附近，住录像店的附近会被录像店吸干我所在地的气运，这人是不是骗子啊？

**Comment by KELERSAMA** [comment]
快去治安局报案
